lib/build/ProjectBuilder.js

```diff
@@ -388,17 +388,17 @@ class ProjectBuilder {
 		}));
 	}
 
-	async _executeCleanupTasks() {
+	async _executeCleanupTasks(options) {
 		this.#log.info("Executing cleanup tasks...");
-		await this._buildContext.executeCleanupTasks();
+		await this._buildContext.executeCleanupTasks(options);
 	}
 
 	_registerCleanupSigHooks() {
 		const that = this;
 		function createListener(exitCode) {
 			return function() {
 				// Asynchronously cleanup resources, then exit
-				that._executeCleanupTasks().then(() => {
+				that._executeCleanupTasks({exitCode}).then(() => {
```

Comment:
Any reason the task might need the exact exit code? I'm wondering whether we should rather have a `force` boolean to make the intention more clear. Note that this is [public API](https://sap.github.io/ui5-tooling/stable/api/@ui5_project_build_helpers_TaskUtil.html#registerCleanupTask) provided to custom tasks, so it should be intuitive.

Reply:
Agree with that. I was thinking that we wanted to keep the API more open for changes, but a `force` param seems logical and correct.